doc/sources/algorithms.rst

```diff
@@ -0,0 +1,96 @@
+####################
+Supported algorithms
```

Comment:
There is no information that just DBSCAN, KMeans, LinReg and LogReg algorithms are supported on GPU for scikit-learn now

Reply:
I think here your help is very much needed in filling out such information, if now we want to see it.

Reply:
There is a larger question - how we would like to present PGU/CPU supported cases? 

Adding new column - might be not sufficient as we will have different params/functionality/data support for CPU and GPU workloads.

Adding new table for GPU? In this case we will have some data duplication.  Are we ok with this?

Reply:
I think such table will look great in the **GPU support** tab

Reply:
@napetrov, can we merge this PR and add GPU-supported algorithms in another PR?

Reply:
yes